packages/protocol/contracts/verifiers/SgxVerifier.sol

```diff
@@ -70,7 +70,6 @@ contract SgxVerifier is EssentialContract, IVerifier {
     error SGX_DELETE_NOT_AUTHORIZED();
     error SGX_INVALID_ATTESTATION();
     error SGX_INVALID_INSTANCE();
-    error SGX_INVALID_INSTANCES();
```

Comment:
In this file also just some very small simplifications and removed (to me) some unnecessary checks.